.github/ISSUE_TEMPLATE/OTHER.yml

```
@@ -0,0 +1,22 @@
+name: Other
+description: Describe anything here.
+title: Other <subject>
```

```suggestion
title: [OTHER] <subject>
```

Let's use the same format for all templates